nh_eobs/tests/report_nh_clinical_observation_report/test_report_without_dob.py

```diff
@@ -20,6 +20,9 @@ def new_patient_pool_mock_patient(*args, **kwargs):
 
         self.patient_pool._revert_method('read')
         self.patient_pool._patch_method('read', new_patient_pool_mock_patient)
+    #
+    # def tearDown(self):
+    #     super(TestObsReportNoDOB, self).setUp()
```

Comment:
Remove this and add a comment explaining that the patch is torn down in the `super`.